### DISCUSSION TITLE
GSoC2024: Reaching out to mentors of pytorch-model-optimizations-with-torchcompile-openvino-backend

### DISCUSSION THREAD

**Discussion by SanftMonster** [openvinotoolkit/openvino#23128] [Google Summer of Code]
Dear OpenVINO developers,
I'd like to participate in the OpenVINO Toolkit's GSoC program. I've made some contributions to open-source projects related to AI frameworks in the past and was fascinated by it. I hope to further learn about OpenVINO and make contributions in GSoC-2024. 

I'd like to choose [pytorch-model-optimizations-with-torchcompile-openvino-backend](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#11-pytorch-model-optimizations-with-torchcompile-openvino-backend) because I have a lot of interest in large models such as LLaVA and CodeLLaMA and accelerating them. Besides, I had some experience with torch.fx (see [one-fx](https://github.com/Oneflow-Inc/one-fx)), which is one of the basic components of `torch.compile`. If given the opportunity to work on this project, I will give my utmost effort to it.

I have already completed the code for my first "good first issue". To submit a well-prepared application, I sincerely wish to get in touch with the project mentors and would appreciate their advice on how to best prepare.

Thank you a lot for your assistance!

**Comment by mlukasze** [comment]
@suryasidd & @ravi9 fyi

**Reply by SanftMonster** [reply]
Thank you!

**Reply by ravi9** [reply]
Hi @AsakusaRinne, Thank you for your interest and completing the Good First Issue. 
Could you kindly share your PR for the good first issue.

Please email me your "application" for review. See [Application Template](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#application-template). 
We can provide any feedback and you could submit it to GSoC.
Please email me with any specific questions you might have.

Cheers,
Ravi.

**Reply by SanftMonster** [reply]
@ravi9 Nice to meet you! My PRs for the good first issue are #23118 and #23174. I'll send you the draft application next week and look forward to your reply.😊

**Reply by ravi9** [reply]
Great ! I look forward to your application :)
